examples/binance-chain/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { render } from 'react-dom'
+import {
+  BinanceBscConnector,
+  UserRejectedRequestError,
+} from '@binance-chain/bsc-connector'
+import { ConnectionRejectedError, UseWalletProvider } from 'use-wallet'
+
+function App() {
+  return (
+    <div>
+      <h1>Binance Chain Connector</h1>
+      <button>Connect</button>
```

Comment:
how onClick could be implemented?

Reply:
I was waiting for the extension to be available so I can test :smile: